pyanaconda/modules/payload/payload_interface.py

```diff
@@ -27,9 +28,18 @@
 class PayloadInterface(KickstartModuleInterface):
     """DBus interface for Payload module."""
 
-    def CreateLiveOSHandler(self):
-        """Create Live OS payload handler and publish it on dbus.
+    def CreateDNFHandler(self) -> ObjPath:
```

Comment:
I think that we should have only one method `CreatePayloadHandler` with a type of the requested handler as an argument.

Reply:
Fixed.